README.md

```diff
@@ -1 +1,39 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+ Description of the project: 
+    This project represents the mobile version of my portfolio where all my projects and skills will be uploaded on it in the future.
+
+
+## Built With
+
+- Major languages: HTML, CSS
+- linter for code analysis
+
+
+## Authors
+
+👤 **Mohammed Elseddik Rebbouh**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] The contact links are broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.